packages/vite/src/node/build.ts

```diff
@@ -502,22 +502,36 @@ async function doBuild(
       }
 
       const watcherOptions = config.build.watch
+      const resolvedChokidarOptions = {
+        ignoreInitial: true,
+        ignorePermissionErrors: true,
+        ...watcherOptions.chokidar,
+        ignored: [
+          '**/node_modules/**',
+          '**/.git/**',
+          ...(watcherOptions?.chokidar?.ignored || [])
+        ]
+      }
+
+      if (isWSL2 && resolvedChokidarOptions.usePolling === undefined) {
+        config.logger.warn(
+          colors.yellow(
+            colors.bold(`(!) `) +
+              'Default file system watching might not work with your setup due to the limitation of WSL2. ' +
```

Comment:
```suggestion
              'Default file system watching might not work with your setup due to a limitation of WSL2. ' +
```

I feel like `a` flows a bit better.